backend/core/settings/base.py

```diff
@@ -13,6 +13,8 @@
 
 DEBUG = os.environ.get('DEBUG') != 'False'
 
+PRODUCTION_URL = 'https://getit.iiitdmj.ac.in'
```

Comment:
imo a env variable is a better option
also what do you mean by production url? let's change it to client_url or frontend_url for now

Reply:
also we should change the variable name

Reply:
oh right got it